chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -59,6 +59,8 @@
 if crowbar_key != ""
   append_line += " crowbar.install.key=#{crowbar_key}"
 end
+# workaround broken IPMI/BMC firmwares not transmitting traffic for 35s (bsc#927997)
+append_line += " netwait=60"
```

Comment:
:-1: there is still some problem here, as in testing, it had many netwait entries in the resulting file - one from each chef-client run.